tests/flow/test_graph_create.py

```diff
@@ -101,7 +101,9 @@ def test07_create_invalid_complex_type_in_array(self):
         queries = ["CREATE (a), (b) SET a.v = [b]",
                    "CREATE (a {v: ['str', [1, NULL]]})",
                    "CREATE (a {v: [[{k: 'v'}]]})",
-                   "CREATE (a:L)-[e:R]->(:L {v: [e]})"]
+                   "CREATE (a:L)-[e:R]->(:L {v: [e]})",
+                   "CREATE (a), (b)-[:R {k:properties(a)}]->(c)",
```

Comment:
what is the returned type `properties` returns ?

Reply:
In that case, ``properties(a)`` returns  an empty map.  The function returns ``T_MAP`` or ``T_NULL``

Reply:
But the function ```properties()``` is not nested within an array, maybe the test should be added in a different function.